server/apps/publish/formatters/newsml_1_2_formatter.py

```diff
@@ -114,7 +114,7 @@ def _format_rights_metadata(self, article, main_news_component):
 
         usage_rights = SubElement(rights_metadata, "UsageRights")
         SubElement(usage_rights, 'UsageType').text = rights['copyrightNotice']
-        SubElement(usage_rights, 'Geography').text = article.get('place', article.get('located', ''))
+        # SubElement(usage_rights, 'Geography').text = article.get('place', article.get('located', ''))
```

Comment:
As per the spec, the definition is different. That's the reason I commented it out. We need to discuss about this.